src/state/initialize/functions/generateTextTriangleParams.js

```diff
@@ -9,7 +9,10 @@ export default (function generateTextTriangleParams(store, viz_area) {
   const text_triangles = {
     row: {},
     col: {},
-    draw: {},
+    draw: {
+      row: [],
+      col: [],
```

Comment:
Does everything still work if these are not set to false by default?

Reply:
yes, it still works.
I have tested under both Chrome & Sarifa.